__tests__/src/core/shuffler.spec.js

```diff
@@ -0,0 +1,28 @@
+import Coords from '../../../src/core/coords';
+import * as Shuffler from '../../../src/core/shuffler';
+
+const mockMathRandom = val => {
+    const mockMath = Object.create(global.Math);
+    mockMath.random = () => val;
+    global.Math = mockMath;
```

Comment:
It's a bad thing to mock globals... It can break others tests in our test suite.
You can use "spyOn" from Jest to mock it in "before" and restore it in "after".

eg: jest.spyOn(global.Math, 'random').mockImplementation(() => 42);